lib/disc.rb

```diff
@@ -110,10 +119,14 @@ def disc_options
       end
 
       def queue
-        disc_options.fetch(:queue, 'default')
+        defined?(@queue) ? @queue : Disc.default_queue
```

Comment:
I'd prefer to keep the `fetch` here (regardless of having a `Disc.default_queue` or not).